docs/security/03-04-ingress-egress-traffic.md

```diff
@@ -74,43 +76,43 @@ The order differs depending on the mode:
   Bring Your Own Certificate
   </summary>
   | **Kind** | **Name** | **Namespace** |
-  | :--- | :--- | :--- | 
+  | :--- | :--- | :--- |
   | Secret | ingress-tls-cert | `kyma-system` |
-  | ConfigMap | net-global-overrides | `kyma-installer `| 
+  | ConfigMap | net-global-overrides | `kyma-installer `|
   | Secret | kyma-gateway-certs | `istio-system` |
   | Secret | kyma-gateway-certs-cacert | `istio-system `|
-  | Secret | apiserver-proxy-tls-cert | `kyma-system` | 
-  | ConfigMap | apiserver-proxy | `kyma-system `|  
+  | Secret | apiserver-proxy-tls-cert | `kyma-system` |
+  | ConfigMap | apiserver-proxy | `kyma-system `|
   </details>
   <details>
   <summary label="demo-xip">
   Demo xip.io setup
   </summary>
   | **Kind** | **Name** | **Namespace** |
-  | :--- | :--- | :--- | 
+  | :--- | :--- | :--- |
   | Secret | ingress-tls-cert | `kyma-system` |
-  | ConfigMap | net-global-overrides | `kyma-installer `| 
+  | ConfigMap | net-global-overrides | `kyma-installer `|
   | Secret | kyma-gateway-certs | `istio-system` |
   | Secret | kyma-gateway-certs-cacert | `istio-system `|
-  | Secret | apiserver-proxy-tls-cert | `kyma-system` | 
-  | ConfigMap | apiserver-proxy | `kyma-system `|  
+  | Secret | apiserver-proxy-tls-cert | `kyma-system` |
+  | ConfigMap | apiserver-proxy | `kyma-system `|
   </details>
   <details>
   <summary label="gardener">
-  Gardener-managed 
+  Gardener-managed
   </summary>
   | **Kind** | **Name** | **Namespace** |
-  | :--- | :--- | :--- | 
+  | :--- | :--- | :--- |
   | Secret | ingress-tls-cert | `kyma-system `|
-  | ConfigMap | net-global-overrides | `kyma-installer `| 
+  | ConfigMap | net-global-overrides | `kyma-installer `|
   | Secret | kyma-gateway-certs-cacert | `istio-system` |
   | Certificate | kyma-tls-cert | `istio-system`|
-  | Certificate | apiserver-proxy-tls-cert | `kyma-system` | 
+  | Certificate | apiserver-proxy-tls-cert | `kyma-system` |
   |ConfigMap | apiserver-proxy | `kyma-system` |
    </details>
 </div>
 
 ## Egress
 Currently no Egress limitations are implemented, meaning that all applications deployed in the Kyma cluster can access outside resources without limitations.
 
->**NOTE:** in the case of connection problems with external services it may be required to create an [Service Entry](https://istio.io/latest/docs/reference/config/networking/service-entry/) object to register the service. 
+>**NOTE:** in the case of connection problems with external services it may be required to create an [Service Entry](https://istio.io/latest/docs/reference/config/networking/service-entry/) object to register the service.
```

Comment:
```suggestion
>**NOTE:** in the case of connection problems with external services it may be required to create a [Service Entry](https://istio.io/latest/docs/reference/config/networking/service-entry/) object to register the service.
```